contrib/fwupd-efi.spec.in

```diff
@@ -0,0 +1,68 @@
+%define alphatag                #ALPHATAG#
+
+%global enable_ci 0
+%global debug_package %{nil}
+
+Summary:   Firmware update daemon EFI binaries
+Name:      fwupd-efi
+Version:   #VERSION#
+Release:   0.#BUILD#%{?alphatag}%{?dist}
+License:   LGPLv2+
+URL:       https://github.com/fwupd/fwupd-efi
+Source0:   http://people.freedesktop.org/~hughsient/releases/%{name}-%{version}.tar.xz
```

Comment:
Shouldn't this be a Github URI?

Reply:
I suppose it *could* be.

Reply:
Hmm, github names the tarballs 1.0.tar.gz -- which isn't awesome. If we can get it to spit out fwupd-efi-1.0.tar.xz I think we can change it then.